HTAN.model.csv

```diff
@@ -64,12 +64,40 @@ Imaging Level 1,Raw imaging data,,"Component, Filename, File Format, HTAN Parent
 Imaging Level 2,Raw and pre-processed image data,,"Component, Filename, File Format, HTAN Participant ID, HTAN Parent Biospecimen ID,  HTAN Data File ID, Channel Metadata Filename, Imaging Assay Type, Protocol Link, Software and Version, Microscope, Objective, NominalMagnification, LensNA, WorkingDistance,WorkingDistanceUnit, Immersion, Pyramid, Zstack, Tseries, Passed QC, Comment, FOV number, FOVX, FOVXUnit, FOVY, FOVYUnit, Frame Averaging, Image ID, DimensionOrder, PhysicalSizeX, PhysicalSizeXUnit, PhysicalSizeY, PhysicalSizeYUnit, PhysicalSizeZ, PhysicalSizeZUnit, Pixels BigEndian, PlaneCount, SizeC, SizeT, SizeX, SizeY, SizeZ, PixelType",,FALSE,Assay,Imaging Level 1,,
 Imaging Level 3 Segmentation,Object segmentations,,"Component, Filename, File Format, HTAN Parent Data File ID,  HTAN Data File ID, Imaging Segmentation Data Type, Parameter file, Software and Version, Commit SHA, Imaging Object Class, Number of Objects",,FALSE,Assay,Imaging Level 2,,
 Imaging Level 3 Image,Quality controlled imaging data,,"Component, Filename, File Format, HTAN Parent Biospecimen ID, HTAN Parent Data File ID, HTAN Parent Channel Metadata ID, HTAN Data File ID, Imaging Assay Type, Protocol Link,Software and Version, Microscope, Objective, NominalMagnification, LensNA, WorkingDistance, Immersion, Pyramid, Zstack, Tseries, Passed QC, Comment, FOV number, FOVX, FOVY, Frame Averaging",,FALSE,Assay,"Imaging Level 3 Channels, Imaging Level 2",,
+Visium RNA-seq Level 1,Files contain raw RNA-seq data associated with spot/slide data.,,"Component, Filename, Run ID, File Format, HTAN Data File ID, HTAN Parent Biospecimen ID, Read Indicator, Spatial Read1, Spatial Read2, Spatial Library Construction Method, Library Preparation Days from Index, Sequencing Library Construction Days from Index, End Bias, Reverse Transcription Primer, Sequencing Platform, Capture Area, Protocol Link, Slide Version, Slide ID, Image Re-orientation, Permeabilization Time, RIN, DV200
```

Comment:
Possibly for the RFC, but are we happy with the component name `Visium RNA-seq` - would `10x Visium Spatial Transcriptomics` be more informative?